index.js

```diff
@@ -1,121 +1,74 @@
-'use strict';
 const request = require('request-promise-native');
 
 const role_to_use = process.env.ROLE;
 const couch_url = process.env.COUCH_URL;
-let compiledUrl = {};
```

Comment:
It's better to create variables with small scope, than reuse a global scope one. If things go wrong, especially with async code, one function could change the URL and break the script in weird, non-deterministic ways.